src/Aspire.Hosting/IServiceMetadata.cs

```diff
@@ -25,3 +25,12 @@ public interface IServiceMetadata : IResourceAnnotation
     /// </summary>
     public string ProjectPath { get; }
 }
+
+internal class ServiceMetadata(string projectPath) : IServiceMetadata
+{
+    public string AssemblyName => throw new NotImplementedException();
```

Comment:
Wondering whether we can ditch assembly name and assembly path now given that we don't auto name components.

Reply:
Looks like we don't reference them anymore. I think we initially had then because we were doing a dotnet run against the assembly itself vs. launching the project. That changed but this never got removed and the assembly name was only ever used as an input for autonaming projects which is now gone.

Reply:
I know I used them somewhere by newing up an instances of the generated classes to get the paths but I can't remember where. No doubt we'll remove it and then I'll find it/remember again :smile: